packages/dbml-core/src/model_structure/endpoint.js

```diff
@@ -55,13 +63,22 @@ class Endpoint extends Element {
       relation: this.relation,
     };
   }
+  
+  setFields (fieldNames, table) {
+    if (typeof fieldNames === "string") fieldNames = [fieldNames] 
```

Comment:
`fieldNames` are always array as the output from pegjs parser so the checking here is not necessary anymore.